x-pack/test/api_integration/deployment_agnostic/apis/observability/alerting/custom_threshold/p99_pct_fired.ts

```diff
@@ -81,8 +83,14 @@ export default function ({ getService }: FtrProviderContext) {
     });
 
     after(async () => {
-      await supertest.delete(`/api/alerting/rule/${ruleId}`).set(internalReqHeader);
-      await supertest.delete(`/api/actions/connector/${actionId}`).set(internalReqHeader);
+      await supertest
+        .delete(`/api/alerting/rule/${ruleId}`)
+        .set(roleAuthc.apiKeyHeader)
+        .set(internalReqHeader);
+      await supertest
+        .delete(`/api/actions/connector/${actionId}`)
+        .set(roleAuthc.apiKeyHeader)
+        .set(internalReqHeader);
```

Comment:
`supertest` is already authenticated as `operator` user, `roleAuthc.apiKeyHeader` will be ignored. Ideally you need to use `supertestWithoutAuth` here

Reply:
Fixed in [a140e02](https://github.com/elastic/kibana/pull/198691/commits/a140e0260823ff8dd463c05969f632a320562647)